src/main/resources/config.yml

```
@@ -8,4 +12,8 @@ messages:
   pvp_disabled: "<red>PVP consent disabled."
   on_cooldown: "<red>You must wait %time% before toggling PVP again."
   pvp_not_consented_attacker: "<red>You tried to hit %player% but PVP is not consented."
-  pvp_not_consented_defender: "<red>%player% tried to hit you but PVP is not consented."
\ No newline at end of file
+  pvp_not_consented_defender: "<red>%player% tried to hit you but PVP is not consented."
+  no_permission: "<red>You don't have permission to use this command."
+  pvp_death_toggle: "<green>PVP disable on death is now %status%."
+  pvp_disabled_on_death: "<red>Your PVP has been disabled due to your death."
+  pvp_status: "<white>Your PVP status is currently <green>%status%<white>."
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)